pnpm-lock.yaml

```diff
@@ -4873,7 +4875,7 @@ packages:
   /@typescript-eslint/typescript-estree/2.27.0:
     dependencies:
       debug: 4.1.1
-      eslint-visitor-keys: 1.3.0
+      eslint-visitor-keys: 1.1.0
```

Comment:
Why was this downgrade required?

Reply:
The lock file changes automatically once you run ./scripts/install.sh, and since there were no changes made to the eslint config manually I confirmed linting still works as expected by running 
```
pnpm recursive run format
```